include/deal.II/grid/tria.h

```diff
@@ -1532,7 +1532,11 @@ class Triangulation : public Subscriptor
      * automatically generated destructor would have a different one due to
      * member objects.
      */
+#ifdef __INTEL_COMPILER
+    virtual ~DistortedCellList() noexcept override {}
+#else
     virtual ~DistortedCellList() noexcept override = default;
+#endif
```

Comment:
Let's just make this change unconditionally. We're going to scratch our heads in a couple of years what we were working around with this. If you wanted to be super clean, you'd just do
```
virtual ~DistortedCellList() noexcept override;
```
in the `.h` file, and 
```
DistortedCellList::~DistortedCellList() noexcept = default;
```
in the `.cc` file.

Reply:
I followed that idea.